src/main/java/entity/room/BlackjackRoomFactory.java

```diff
@@ -0,0 +1,17 @@
+package entity.room;
+
+import java.io.IOException;
+import java.util.List;
+
+import entity.player.Player;
```

Comment:
Is there a need for this Factory? Would we have any case where we're just instantiating abstract rooms and delegating to a Factory to figure out which type? In CA we have some many split up Interactors that I think we always have enough context to instantiate a BlackjackRoom explicitly and store it in a Game variable if we need polymorphism.

Reply:
you have a point, I've turned the factory into a generic one just like the player factory. It's still a bit "redundant" I guess for now if we have just one game blackjack. We can decide on whether to keep the generic factory later.